draft-ietf-dnsop-svcb-https.md

```diff
@@ -770,8 +770,30 @@ ALPN IDs that do not uniquely identify a protocol suite (e.g. an ID that
 can be used with both TLS and DTLS) are not compatible with this
 SvcParamKey and MUST NOT be included in the ALPN set.
 
+To establish a connection to the endpoint, clients MUST
+
+1. Identify all supported IDs in the ALPN set.
+2. Collect the set of transports (e.g. TLS, DTLS, QUIC) implied by those IDs.
+3. For each transport in this set, construct a ProtocolNameList containing all
+the client's supported IDs for that transport, without regard to the ALPN set
+from SVCB.
+
+For example, if the ALPN set is \["http/1.1", "h3"\], the client could attempt
```

Comment:
Might want to mention in this example that this client supports ["http/1.1", "h2", "h3"]. If the client supports "h4" which also goes over QUIC, the example would be different.

Reply:
Done